discussion.txt

```diff
@@ -0,0 +1,3 @@
+It isn't safe because the client-side is easier to be attacked, the server doens't have control over the client 
+environment, and can be target of SQL injection, cross-site scripting (XSS), and other forms of attacks. Also, the browsers
+may handle in different ways with Javascript, resulting on inconsistent validation.
```

Comment:
I wanted to see specific mention of the fact that client-side validation is easily bypassed using tools like the browser dev tools